src/main/java/ch/cern/eam/wshub/core/services/INFOR_OPERATION.java

```diff
@@ -134,6 +134,14 @@ public enum INFOR_OPERATION {
     EQP_REPL("EQP_REPL"),
     EQP_GRAPH_R("EQP_GRAPH_R"),
     EQP_CAMP_C("EQP_CAMP_C"),
+
+    // NONCONFORMITIES
+    NONCONF_C("NONCONF_C"),
+    NONCONF_R("NONCONF_R"),
+    NONCONF_R_DEF("NONCONF_R_DEF"),
```

Comment:
The other instances of read default web-services append _RD. Since it is a small thing, I will accept and change it manually afterwards